paper.md

```diff
@@ -1,33 +1,28 @@
 ---
-title: 'Separation using ChatGPT'
+title: 'Benchmarks and leaderboards for sound demixing tasks'
 tags:
   - separation
-  - u-net
+  - leaderboards
+  - benchmarks
 authors:
-  - name: Fabian-Robert Stoter^[co-first author] # note this makes a footnote saying 'co-first author'
-    orcid: 0000-0003-0872-7098
-    affiliation: "1, 2" # (Multiple affiliations must be quoted)
-  - name: Author Without ORCID^[co-first author] # note this makes a footnote saying 'co-first author'
+  - name: Roman Solovyev^[co-first author] # note this makes a footnote saying 'co-first author'
+    orcid: 0000-0003-0312-452X
+    affiliation: 1
+  - name: Alexander Stempkovskiy
+    affiliation: 1
+  - name: Tatiana Habruseva^[corresponding author]
+    orcid: 0000-0003-3940-8639
     affiliation: 2
-  - name: Author with no affiliation^[corresponding author]
-    affiliation: 3
 affiliations:
- - name: Audioshake
+ - name: Institute for Design Problems in Microelectronics of Russian Academy of Sciences
    index: 1
- - name: Institution Name
-   index: 2
  - name: Independent Researcher
-   index: 3
-date: 10 September 2023
+   index: 2
+date: 09 October 2023
 bibliography: paper.bib
-arxiv-doi: 10.21105/joss.01667
+arxiv-doi: 10.48550/arXiv.2305.07489
```

Comment:
@ZFTurbo we decided to not use the arxiv links on the side of the pdf so please make sure that your paper is cited inline

Reply:
Ok. But I didn't understand if some action required from me now?

Reply:
Sorry, I meant: if you want to cite your paper please mention it inline using the markdown syntax such as [@paperkey]

Reply:
@ZFTurbo can you please add the reference to your submission by tomorrow?

Reply:
> @ZFTurbo can you please add the reference to your submission by tomorrow?

I didn't understand what I need to do. Can you please explain in more details.

Reply:
@ZFTurbo sorry for not being precise. I would suggest you 

1. add your paper to the bibtex references

```
@misc{solovyev2023benchmarks,
      title={Benchmarks and leaderboards for sound demixing tasks}, 
      author={Roman Solovyev and Alexander Stempkovskiy and Tatiana Habruseva},
      year={2023},
      eprint={2305.07489},
      archivePrefix={arXiv},
      primaryClass={cs.SD}
}
```

2. cite the paper in the abstract using `[@solovyev2023benchmarks]` instead of using the doi value

Reply:
I did update. I hope I understood correctly )

Reply:
Almost. Let me fix that with a PR ;)

Reply:
@ZFTurbo i did a proposal for changes in #13. Please have a look there, maybe then its more clear what I meant. Also, please check the rendering on the urls, this doesn't look well formatted, you may just use a different bibtex entry so that it doesn't show the full url

<img width="706" alt="image" src="https://github.com/sdx-workshop/sdx-submissions/assets/72940/c4a5d8e4-608e-4f2b-8b7a-8b4de98f2321">